.circleci/config.yml

```diff
@@ -16,12 +16,24 @@ jobs:
           - v2-dependencies-{{ checksum "Gemfile.lock" }}
           # fallback to using the latest cache if no exact match is found
           - v2-dependencies-
+
       - run:
           name: install dependencies
-          command: bundle install --jobs=4 --retry=3 --path vendor/bundle
+          command: bundle install --jobs=4 --retry=3
```

Comment:
Based on my understanding of the cache config before and after this, it seems like dropping `--path vendor/bundle` will prevent CircleCI from caching these dependencies across builds — am I missing something?

Reply:
Oooh, good comment and question! Let me take a look at this...

Reply:
Ah, interesting! I wonder if calling `bundle exec jekyll build` would solve the issue. 

I’m not sure where the default installation folder for bundler is — perhaps we could just directly cache that folder instead. 

Also, what folder it uses isn’t really important either, and if `vendor/bundle` says something else to you, let’s change it! In [18F/identity-style-guide](https://github.com/18F/identity-style-guide/blob/8090b6a26eb3d07c22c3939a1d16bb7c65fa06bf/.circleci/config.yml#L11-L23) I went with `~/.bundler` to mirror `~/.npm`, for example.

Reply:
On a more meta-level, if you haven’t seen the [CircleCI local command line tools](https://circleci.com/docs/2.0/local-cli/), they might be helpful in helping to more quickly try out a configuration without having to wait for a push + build on CI — I just learned about it last month!

Reply:
@adunkman Oooooooh! Taking a look at these local CLI tools! 👀👀

Reply:
@adunkman `bundle exec` solved the issue ... `circleci local execute` got it done without a bunch of test pushes to GitHub ... thank you for your help on both of those! 

Now all that's needed is one last pair of 👀 and a ✅!

Reply:
it LGTM, but I hesitate to hit "approve" since last time I did @adunkman immediately found an issue! So I'll leave it to him :-)

Reply:
@adunkman the journey was the destination all along! it's what we learned along the way!